### PR TITLE
Issue #196 ... implement 1-step Poliphon

### PR DESCRIPTION
**Pull request by AthenaAugousta**
Hey @HolgerPollyNet, could you check how it performs now?

### CODE REVIEW

File: lib/retrievals/poliphon_one.m [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+                err_aerBsc1064_raman_nd1(k,i) = 0.15*aerBsc1064_raman_nd1(k,i);
+            end
+        end
+    end
+end
+
+POLIPHON1 = struct();
+POLIPHON1.aerBsc355_klett_d1 = aerBsc355_klett_d1;
+POLIPHON1.aerBsc355_klett_nd1 = aerBsc355_klett_nd1;
+POLIPHON1.aerBsc532_klett_d1 = aerBsc532_klett_d1;
```
Comment (ZPYin) [inline lib/retrievals/poliphon_one.m:198]: The error should be caused by the `for` loops above. The variables ending with `_nd1` or `d1` can be undefined, if `aerBscxxx_raman` or `aerBscxxx_klett` is empty in the second dimension (no valid retrieved profiles).

I suggest to pre-define the output variables first and then filling them with some target values, which can avoid many troubles (see below):

```matlab
% pre-allocated variables
aerBsc355_klett_d1 = NaN(size(aerBsc355_klett));

% filling with values
for iH = 1:size(aerBsc355_klett, 1)
    for iT = 1:size(aerBsc355_klett, 2)
        aerBsc355_klett_d1(iH, iT) = aerBsc355_klett(iH, iT);
    end
end
```

Comment (HolgerPollyNet) [reply lib/retrievals/poliphon_one.m:198]: this was my idea also, thanks @ZPYin ! So please include the variable initiation and then the problem should be fixed! @AthenaAugousta

**Comment by HolgerPollyNet** [comment]
Good news. With the latest version, the chain is running without problems and nc files are created!

Great!!!

Now the picky things:
Could you please homogenize the variable names in the output? Currently we use uncertainty_*** for the error estimates but you used err_***.

Could you rename it? At a certain point we may think about the names in general, but for now it would be great to keep some consistency.

**Comment by AthenaAugousta** [comment]
That's great to hear! 

Thanks for pointing out the naming inconsistency, I will change it accordingly. Also, if you find something else that can be improved, please let me know.

**Comment by HolgerPollyNet** [comment]
Congratulations, the new commit worked without problems! @AthenaAugousta 

You did a great job and accomplished to implement a new product to the PPC. This qualifies you to give the next PollyNET webinar under the tentative title: Adding new products/features to PollyNET Processing Chain - experiences from a newcomer" ;-)  - which I think would be really warmly welcomed by all the other who want to contribute but do not yet really know how to start!

**Pull request merged**